### PR TITLE
Feature/css carousel migrate to hooks and typescript

### PR DESCRIPTION
**Pull request by hacheG**

### CODE REVIEW

**Review by cazapatamar — CHANGES_REQUESTED** [review]

File: src/pages/home/content/CssCarousel.tsx [hunk 1]
```diff
@@ -46,17 +66,26 @@ function getNextElement(list) {
   return null;
 }
 
-function usePosition(ref, moreThan4) {
-  const [prevElement, setPrevElement] = React.useState(null);
-  const [nextElement, setNextElement] = React.useState(null);
+interface RefType {
+  current: HTMLDivElement;
+}
```
[the remark this answers is not in the document]
Comment (hacheG) [reply src/pages/home/content/CssCarousel.tsx]: done

File: src/pages/home/content/CssCarousel.tsx [hunk 2]
```diff
@@ -3,9 +3,29 @@
  */
 
 /* eslint-env browser */
```
[the remark this answers is not in the document]
Comment (hacheG) [reply src/pages/home/content/CssCarousel.tsx]: deleted

File: src/pages/home/content/CssCarousel.tsx [hunk 3]
```diff
@@ -26,17 +49,17 @@ const Container = styled.div`
...
+function getPrevElement(list: Array<Element>): HTMLElement | null {
   const sibling = list[0].previousElementSibling;
 
   if (sibling instanceof HTMLElement) {
     return sibling;
   }
 
-  return sibling;
+  return null;
 }
```
Comment (cazapatamar) [inline src/pages/home/content/CssCarousel.tsx:60]: Please migrate all regular functions to arrow functions

**Review by cazapatamar — CHANGES_REQUESTED** [review]

**Review by cazapatamar — APPROVED** [review]

**Comment by hacheG** [comment]
i did a revert because i found a bug on content component

**Pull request merged**